src/Orc.Controls/Themes/Generic.generated.xaml

```diff
@@ -2,6 +2,8 @@
 <ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:local="clr-namespace:Orc.Controls" xmlns:catel="http://schemas.catelproject.com" xmlns:automation="clr-namespace:Orc.Controls.Automation" xmlns:sys="clr-namespace:System;assembly=mscorlib" xmlns:converters="clr-namespace:Orc.Controls.Converters" xmlns:xamlbehaviors="http://schemas.microsoft.com/xaml/behaviors" xmlns:orctheming="http://schemas.wildgums.com/orc/theming">
   <ResourceDictionary.MergedDictionaries>
     <ResourceDictionary Source="/Orc.Theming;component/themes/generic.xaml" />
+    <ResourceDictionary Source="/Orc.Controls;component/Controls/DirectoryPicker/Themes/DirectoryPicker.generic.xaml" />
```

Comment:
looks as it something wrong with generated xaml

Reply:
it should not be merged dictionaries ...because generated.xaml itself solves a problem with merged dictionaries